src/main/java/org/spongepowered/common/registry/type/item/InventoryArchetypeRegistryModule.java

```diff
@@ -325,7 +325,7 @@ public void registerDefaults() {
             .with(new SpongeInventoryArchetypeBuilder()
                 .from(MENU_GRID)
                 .property(new InventoryDimensionImpl(5,3))
-                .build("horse_grid", "HorseGrid"))
+                .build("minecraft:horse_grid", "HorseGrid"))
```

Comment:
@Faithcaio Please, review this one.

Reply:
Look good to me

Reply:
actually would be sponge as the archetypes are smth. we add

Reply:
I think I'll leave this as it is. I'd have to change https://github.com/ImMorpheus/SpongeCommon/blob/13cc5d531cf2437b83a009a5efff4db6c5dee703/src/main/java/org/spongepowered/common/registry/type/item/InventoryArchetypeRegistryModule.java#L214 too. Let's wait for the catalog key PR.